### PR TITLE
Fix #113

### LINKED ISSUE #113: `kubecfg init` does not include k.libsonnet
k8.libsonnet (in "/vendor/schema/dev /") is only created with 'kubecfg init' should also include 'k.libsonnet' somewhere, maybe the whole ksonnet.beta.2 folder (?)

### PR DESCRIPTION
**Pull request by hausdorff**

### CODE REVIEW

**Comment by anguslees** [comment]
Please add a description that gives some idea of the change.  A future me that is doing some complicated git history search thanks you.

File: vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/ksonnet/emit.go [hunk 1]
```diff
@@ -13,12 +13,19 @@ import (
 
 // Emit takes a swagger API specification, and returns the text of
 // `ksonnet-lib`, written in Jsonnet.
```
Comment (jessicayuen) [inline vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/ksonnet/emit.go]: Not directly related to this PR, but I was trying to make sense of the dependency changes ... would have been nice to see this comment updated to reflect the new return objects.

File: metadata/manager.go [hunk 2]
```diff
@@ -184,12 +191,12 @@ func (m *manager) createAppDirTree() error {
 	return nil
 }
 
-func generateKsonnetLibData(ksonnetLibDir AbsPath, text []byte) ([]byte, error) {
+func generateKsonnetLibData(ksonnetLibDir AbsPath, text []byte) ([]byte, []byte, error) {
```
Comment (jessicayuen) [inline metadata/manager.go]: nil is always being returned for the second return object it seems

**Comment by hausdorff** [comment]
Added more detailed commit message per @anguslees. Will merge when CI passes.

**Pull request merged**